gcc/rust/ast/rust-ast.h

```diff
@@ -399,7 +399,7 @@ class SimplePathSegment : public PathSegment
   {
     return as_string ().compare ("crate") == 0;
   }
-  bool is_lower_self () const { return as_string ().compare ("self") == 0; }
+  bool is_lower_self_seg () const { return as_string ().compare ("self") == 0; }
   bool is_big_self () const { return as_string ().compare ("Self") == 0; }
```

Comment:
Not entirely sure why `is_lower_self` got renamed, shouldn't `is_big_self` be renamed too ?

Reply:
this is for using them in a templated function, so that the API is consistent with other Path-like nodes. I think changing `is_big_self` also makes sense